test/base/SpecBase.scala

```diff
@@ -60,7 +60,23 @@ trait SpecBase
 
   val defaultBodyParser: PlayBodyParsers = app.injector.instanceOf[PlayBodyParsers]
 
-  val sessionRequest = new FakeSessionRequestAction(emptyUserAnswers)
+  val emptySessionRequest = new FakeSessionRequestAction(emptyUserAnswers)
+
+  private val categorisationAnswers =
+    CategorisationAnswers(Some(CommodityCode("anything")), Some(CountryOfOrigin("GB")))
+
+  private val maintainProfileAnswers =
+    MaintainProfileAnswers(
+      Some(UkimsNumber("anything")),
```

Comment:
should these be valid numbers? do we care for testing purposes?

Reply:
probably but I've not been doing it either. Since we are not testing the values of the field directly it probably doesn't matter right now?